internal/cli/kraft/run/runner_kraftfile_runtime.go

```diff
@@ -354,6 +356,17 @@ func (runner *runnerKraftfileRuntime) Prepare(ctx context.Context, opts *RunOpti
 		machine.Spec.ApplicationArgs = runtime.Command()
 	}
 
+	var kernelArgs []string
+	if (!runtime.KConfig().AllNoOrUnset("CONFIG_LIBUKRANDOM_LCPU") || !cpuid.CPU.Has(cpuid.RDRAND)) &&
```

Comment:
The 1st condition here looks incorrect. It should check if `CONFIG_LIBUKRANDOM_LCPU` is *not* enabled.